generator/src/subcommands/gen_cmake/target.rs

```diff
@@ -184,22 +184,22 @@ impl CargoTarget {
 
             if(archive_byproducts)
                 _corrosion_copy_byproducts(
-                    {target_name} ARCHIVE_OUTPUT_DIRECTORY \"${{cargo_build_out_dir}}\" \"${{archive_byproducts}}\"
+                    {target_name} ARCHIVE_OUTPUT_DIRECTORY \"${{cargo_build_out_dir}}\" \"${{archive_byproducts}}\" FALSE
                 )
             endif()
             if(shared_lib_byproduct)
                 _corrosion_copy_byproducts(
-                    {target_name} LIBRARY_OUTPUT_DIRECTORY \"${{cargo_build_out_dir}}\" \"${{shared_lib_byproduct}}\"
+                    {target_name} LIBRARY_OUTPUT_DIRECTORY \"${{cargo_build_out_dir}}\" \"${{shared_lib_byproduct}}\" FALSE
                 )
             endif()
             if(pdb_byproduct)
                 _corrosion_copy_byproducts(
-                    {target_name} PDB_OUTPUT_DIRECTORY \"${{cargo_build_out_dir}}\" \"${{pdb_byproduct}}\"
+                    {target_name} PDB_OUTPUT_DIRECTORY \"${{cargo_build_out_dir}}\" \"${{pdb_byproduct}}\" FALSE
                 )
             endif()
             if(bin_byproduct)
                 _corrosion_copy_byproducts(
```

Comment:
I reckon this will generally go wrong, as there's no suffix handling on the caller side here. What's the status of this code path?

Reply:
I was planning to remove the legacy generator for v0.5 and require a newer Cmake version (was thinking of 3.22 as in ubuntu 22.04). I could do it now.

Reply:
I’m curious: is the newer cmake version requirement related to the removal of the generator?

Reply:
The legacy Generator enabled CMake support for CMake < 3.19 (3.19 introduced JSON parsing support). There was also some Multi-Config related code which was conditionally enabled based on CMake 3.20 / 3.21, so I decided to declare CMake 3.22 as a minimum supported version to have a bit of wiggle-room, while still supporting ubuntu-22.04 out of the box.

Reply:
Thanks, makes sense to me. Looking forward to the update :)